### PR TITLE
Update mspc.json

### PR DESCRIPTION
**Pull request by VJalili**
The Algolia search on [MSPC website](https://genometric.github.io/MSPC/) does not return any hits (see the screenshot below), in an effort to fix it, updating the search configurations following [algolia docs](https://docsearch.algolia.com/docs/config-file/). 

Screen Shot 2021-08-21 at 1 58 34 PM

### CODE REVIEW

File: configs/mspc.json [hunk 1]
```diff
@@ -41,8 +49,5 @@
       "type"
     ]
   },
-  "conversation_id": [
-    "1181088484"
-  ],
```
Comment (shortcuts) [inline configs/mspc.json:46]: Could you please restore the `conversation_id`?

**Review by shortcuts — COMMENTED** [review]
Let me know if there's still no results after the merge

File: configs/mspc.json (hunk 1 above)
Comment (VJalili) [reply configs/mspc.json:46]: Sure, though I am not sure what is the `conversation_id` and how it is set, is the current value correct? (i.e., `1181088484`)

Comment (shortcuts) [reply configs/mspc.json:46]: Hey, yes it is the correct value. We use it internally to match a config to its owner when they apply for DocSearch.

Comment (VJalili) [reply configs/mspc.json]: Thanks! I reverted that change.

**Comment by VJalili** [comment]
@shortcuts Thank you for merging this; though I am still getting no results.

**Pull request merged**